apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -69,6 +70,24 @@ export const useTableColumns = (sort: Array<string>, filters: IFilters) => {
                 key: 'number',
                 sorter: true,
                 width: '10%',
+                render: (num) => {
+                    const text = String(num)
+                    const search = getFilteredValue(filters, 'search')
+                    if (!isEmpty(search)) {
```

Comment:
Maybe, it is possible to wrap Highlighter in some function in order to avoid copy/paste

Reply:
It's to match copypasta